config/locales/en.yml

```diff
@@ -128,192 +103,3 @@ en:
       <p class="govuk-body">Your session has ended because you have not done anything for 4 hours. You'll have to start again.</p>
 
       <p class="govuk-body">We do this for your security. We've deleted all the details you entered to protect your data.</p>
-  coronavirus_form:
```

Comment:
I wonder if this heading, the "errors" section and an empty "questions" section should be kept in?